packages/web-api-hooks/src/useClipboard.ts

```diff
@@ -0,0 +1,72 @@
+export default function useClipboard() {
+  const checkForPermission = (type: PermissionName): Promise<string> => {
```

Comment:
This function may be put outside of the `useClipboard` function (just above it), in order to avoid re-instantiating it on every render.

Reply:
On a second thought, `checkForPermission` may be moved to 'utils.ts', in order to be [reused](https://github.com/kripod/react-hooks/blob/579f406ac586ac404d99d2cd9ba27931dc6384b3/packages/web-api-hooks/src/useDeviceMotion.ts#L27) by `useDeviceMotion`.

Reply:
Moved.